src/Turbo/Tests/app/templates/books.html.twig

```diff
@@ -3,12 +3,13 @@
 {% block body %}
     <h1>Books</h1>
 
-    <div id="books" {{ turbo_stream_listen('App\\Entity\\Book') }}></div>
+    <div id="books" {{ turbo_stream_listen('books') }}></div>
```

Comment:
What was the significance of this change?

Reply:
It's due to this: https://github.com/symfony/ux/pull/327/files#diff-de5c61126b72f0bca139694b277c0a2ac1cd5ff7b1873a3d695783a4e374e95fR23

I could also fallback on `App\\Entity\\Book` in that else in the expression but this is more in line with `book_by_author_1`.
If there's a preference I can change it.

Reply:
You should keep the current test as-is and create a new one (with a new entity if necessary) to test the new feature.

Reply:
@dunglas Ah yes, of course, otherwise we don't know if BC was broken. 👍